pkg/capture/capture.go

```diff
@@ -216,7 +211,10 @@ func (c *Capture) process() <-chan error {
 	c.wgProc.Add(1)
 	go func() {
 
-		defer c.wgProc.Done()
+		defer func() {
+			close(captureErrors)
```

Comment:
Added the `close()` statement (not only to track errors, but simply because we were missing that part of cleanup entirely so far, which would leave us with dead stuff after each interface teardown).